README.md

```diff
@@ -131,20 +131,20 @@ Adjust the base Sidekiq Job class, often called `ApplicationJob` or similar, so
 ```ruby
 sidekiq_retry_in do |_count, exception, jobhash|
   case exception
-  when Infrastructure::Sidekiq::Disposal::JobKilled
+  when Sidekiq::Disposal::JobKilled
     # Optionally log/collect telemetry here too…
     :kill
-  when Infrastructure::Sidekiq::Disposal::JobDropped
+  when Sidekiq::Disposal::JobDropped
     # Optionally log/collect telemetry here too…
-    :discard
+    :drop
```

Comment:
We actually need `:discard` here as that's [the symbol Sidekiq uses](https://github.com/sidekiq/sidekiq/wiki/Error-Handling#configuration). I'm actually thinking we should change `sidekiq-disposal` to use `:kill` and `:discard` for consistency, rather than having our own `:drop` which is mapped to Sidekiq's `:discard`.

Reply:
ahh right. Yea I think aligning with Sidekiq makes sense here. I'll push up a PR for that instead.